### PR TITLE
Prime CommandGroup

### LINKED ISSUE #105: Add Prime command
Add a new command group to prime the system that backs the ball out to the primed location, and then spins up the shooter wheels
- [x] Add new command for the intake system that moves the ball to the primed location
- [x] Add a command group that calls the above command, and then calls `RunShooterManually` in order to spin up the shooter.

### PR DESCRIPTION
**Pull request by wilstenholme**
Add prime command group to prime boulder by backing boulder out to primed location, then spins up the shooter wheels.
closes #105

### CODE REVIEW

File: src/main/java/org/usfirst/frc/team5687/robot/Constants.java [hunk 1]
```diff
@@ -70,6 +75,18 @@
         public static final double BOWL_SPEED = -1.0;
     }
 
+    public class Shooter {
+        /**
+         * Time for running shooter wheels for priming
+         */
```
Comment (rtfoley) [inline src/main/java/org/usfirst/frc/team5687/robot/Constants.java]: 5s is a long time to wait, is this really the required length of time for the shooter to spin up?

File: src/main/java/org/usfirst/frc/team5687/robot/commands/PrimeBoulder.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    @Override
+    protected void execute() {
+        intake.setSpeed(Constants.Intake.PRIME_SPEED);
+    }
+
+    @Override
+    protected boolean isFinished() {
+        return intake.isPrimeAble();
```
Comment (rtfoley) [inline src/main/java/org/usfirst/frc/team5687/robot/commands/PrimeBoulder.java]: We might want to change the method/ constant named to "primed" vs "primable" for clarity.

File: src/main/java/org/usfirst/frc/team5687/robot/commands/SetShooterSpeed.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package org.usfirst.frc.team5687.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.omg.PortableInterceptor.SYSTEM_EXCEPTION;
```
Comment (rtfoley) [inline src/main/java/org/usfirst/frc/team5687/robot/commands/SetShooterSpeed.java]: Is this import really used?

File: src/main/java/org/usfirst/frc/team5687/robot/commands/SetShooterSpeed.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    @Override
+    protected boolean isFinished() {
+        return System.currentTimeMillis() > endTime;
+    }
+
+    @Override
+    protected void end() {
+        shooter.stop();
```
Comment (rtfoley) [inline src/main/java/org/usfirst/frc/team5687/robot/commands/SetShooterSpeed.java]: Wouldn't this stop the shooter wheels at the end of the priming operation? We want them to continue spinning indefinitely.

**Comment by rtfoley** [comment]
Looks pretty good, let's just change 'primable' to 'primed'

File: src/main/java/org/usfirst/frc/team5687/robot/subsystems/Intake.java [hunk 5]
```diff
@@ -71,16 +71,13 @@ public boolean isPrimeAble() {
     public void updateDashboard() {
         SmartDashboard.putNumber("IR distance", boulderSensor.getValue());
         if (!isDetected()){
-            SmartDashboard.putString("Boulder is", "Not Detected");
-        }
-        else if (!isPrimeAble()){
-            SmartDashboard.putString("Boulder is", "Detected");
```
Comment (rtfoley) [inline src/main/java/org/usfirst/frc/team5687/robot/subsystems/Intake.java:77]: @wilstenholme did you mean to remove this scenario?

Comment (wilstenholme) [inline src/main/java/org/usfirst/frc/team5687/robot/subsystems/Intake.java:77]: @rtfoley yes, it shouldn't need to be there, right? Or should the condition be isDetected() instead?

Comment (rtfoley) [inline src/main/java/org/usfirst/frc/team5687/robot/subsystems/Intake.java:77]: I think it is helpful to keep it and log the isDetected state as well.

Comment (wilstenholme) [inline src/main/java/org/usfirst/frc/team5687/robot/subsystems/Intake.java:77]: Done

**Pull request merged**